docs/cne-install.md

```diff
@@ -0,0 +1,177 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+
+The CNE installer can only upgrade the SAT product stream for this release. See [Install the System Admin Toolkit Product Stream](install.md) for installation instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both automated and logged to help you save time. The CNE installer can be used to upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000) for detailed information about `cne-install` and its options.
+
+### Prerequisites
+
+- CSM is installed and verified.
+- cray-product-catalog is running.
```

Comment:
Since `cray-product-catalog` is a [Kubernetes ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/), I'm not sure "running" is an appropriate verb. I'm struggling at the moment to find a more precise wording, but perhaps "The `cray-product-catalog` has been populated" or "initialized" would make more sense in this context?

Reply:
I agree this is a little awkward, though I would note this is just being copied from `install.md`, the wording is weird in both places.

It could be referring to the helm chart, also named `cray-product-catalog` that creates the config map, though in that case I still don't think "running" is very good, I'd probably prefer "deployed" for talking about a helm chart state.

I'd probably prefer one of:
> The `cray-product-catalog` Kubernetes ConfigMap has been populated.
Or:
> The `cray-product-catalog` Helm chart has been deployed.

Reply:
@ekamin-hpe, like you mentioned, I grabbed this information from the install.md file. I can update in both places, but I'm not actually sure what this should be referring to (the ConfigMap or Helm chart). Is there a way to find this out?

Reply:
Both statements would be correct, because the `cray-product-catalog` Helm chart creates a config map called `cray-product-catalog`. I think it's probably more direct to refer to the config map rather than the helm chart, so I'd prefer the first form I think.

That said, looking again I don't actually like my first suggestion very much. "Populated" implies that there needs to be some specific data in the config map beforehand, which is not the case, it just needs to exist. The install process includes a part that adds data to the config map, so it might be confusing to read it as needing to have some special data in it. We could also say "the `cray-product-catalog` Kubernetes ConfigMap has been created" or something.

I also of doubt how useful it is to name this as a dependency. `cray-product-catalog` is a part of CSM so I don't exactly know why we have been naming it separately.

Reply:
It looks like we discussed this same thing about a month back when we were reviewing the following:
- [STP-2831](https://jira-pro.its.hpecorp.net:8443/browse/STP-2831)
- https://connect.us.cray.com/confluence/display/~ekamin/SAT+documentation+feedback+from+product+owners+and+stakeholders+for+SAT+2.2

I remember discussing if we should remove the second bullet, provide a link to the getting started guide, and maybe add authentication/credentials required to this section. For the purposes of this pull request, would it be better to just remove the second bullet? We can try to improve this section further for Papaya if we have time, but seems like this would resolve the immediate issue.

Reply:
I'd be good with removing it in the short term, although for consistency it would be nice to remove it from both places (i.e. here and `install.md`). @jack-stanek-hpe and @haasken-hpe how would you feel about removing this prerequisite? I personally think it's redundant at best and at worst is misleading.

Longer term I honestly don't think we would need to refine it any more than that, though if there was an appropriate place to put another reference to the GSG that would obviously be fine. I honestly just think this particular bullet point is kind of pointless.

Reply:
I'm in favor or removing this second bullet point.